src/sources/eventstoredb_metrics/mod.rs

```diff
@@ -110,15 +115,14 @@ fn eventstoredb(
 
                             Ok(stats) => {
                                 let metrics = stats.metrics(namespace.clone());
+                                let count = metrics.len();
+                                let byte_size = bytes.len();
```

Comment:
Same here, this should be using `BytesSizeOf::size_of` to calculate the in-memory byte count.